### DISCUSSION TITLE
Problem connecting Standalone Topic Operator  to confluent.cloud

### DISCUSSION THREAD

**Discussion by jvieten** [strimzi/strimzi-kafka-operator#7956] [General]
Hello.
I would like to runthe  Standalone Topic Operator against an managed confluent cloud instance.

 I can only provide a STRIMZI_KAFKA_BOOTSTRAP_SERVERS Url.

But the STRIMZI_ZOOKEEPER_CONNECT env variable  is mandatory.

CAn anybody help?

**Comment by scholzj** [comment]
The Topic Operator requires ZooKeeper connection. That is why the environment variable is mandatory ;-).

**Reply by jvieten** [reply]
Ah Ok so its not possible to use together with a managed confluent cloud instance because they do not open the ZooKeeper Url.
